packages/eslint-plugin-query/src/__tests__/exhaustive-deps.test.ts

```diff
@@ -679,33 +679,6 @@ ruleTester.run('exhaustive-deps', rule, {
         },
       ],
     },
-    {
-      name: 'should fail when a queryKey is a reference of an array expression with a missing dep',
```

Comment:
why was this test removed?

Reply:
typescript-eslint now detects duplicated test cases (https://github.com/typescript-eslint/typescript-eslint/issues/9018) so this was failing after upgrading.

I don't know why it was duplicated in the first place.

https://github.com/TanStack/query/blob/f353ef8d3d3973ac6de8155eb69cb943dcdbb8df/packages/eslint-plugin-query/src/__tests__/exhaustive-deps.test.ts#L655-L708